docs/index.md

```diff
@@ -1,8 +1,75 @@
 # Zampy
 
-A tool for downloading Land Surface Model input data.
+A tool for downloading Land Surface Model (LSM) input data.
 
 Named after *Zam*; [the Avestan language term for the Zoroastrian concept of "earth"](https://en.wikipedia.org/wiki/Zam).
 
+## Why using Zampy
+`zampy` is designed to retrieve data for LSM model input. It can help you prepare the data within the following steps:
+
+1. **Download** the data for the specified location(s) / geographical area.
+2. **Ingest** data into unified (zampy) format.
+3. **Load** the variables in a standardized way (standardized names & standardized units).
+4. **Convert** the data to standard formats:
+    - ALMA / PLUMBER2's ALMA formatted netCDF.
+    - *CMOR formatted netCDF*.
+
+(Note: items in *italic* will not be worked on for now/low priority, but we want to allow space for these in the future.)
+
 ## How to use Zampy
-See the section ["using Zampy"](using_zampy.md). Information on the available datasets and their variables is shown [here](available_datasets.md).
+
+To install `zampy`, check [this guide](configuration.md). Please also make sure that you have it properly configured following the guidelines.
+
+We recommend our users to use `zampy` with recipes.
+
+A "recipe" is a file with yml extension, it defines:
```

Comment:
This list is broken in the docs preview. You need an empty line before the list.
```suggestion
A "recipe" is a file with yml extension, it defines:

```